motr/io_nw_xfer.c

```diff
@@ -921,6 +1004,19 @@ static int target_ioreq_iofops_prepare(struct target_ioreq *ti,
 
 	ndom_max_segs = m0_net_domain_get_max_buffer_segments(ndom);
 
+	num_units = (filter == PA_PARITY) ? 
+					ti->ti_cksum_data[M0_PUT_PARITY].cd_num_units :
+					ti->ti_cksum_data[M0_PUT_DATA].cd_num_units;
+
+	// Compute how many segment are per unit
+	if( m0__obj_is_di_enabled(ioo) && num_units &&
+	   (ioo->ioo_oo.oo_oc.oc_op.op_code == M0_OC_WRITE ||
+	    ioo->ioo_oo.oo_oc.oc_op.op_code == M0_OC_READ) )  {
```

Comment:
M0_IN().

Reply:
We have removed this part of code long back, looks like github not rendered latest code.